loudness/patterns/front-page.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Title: Front Page
+ * Slug: loudness/front-page
+ */
+?>
+
+<?php
+if ( 'posts' === get_option( 'show_on_front' ) ) {
```

Comment:
This is clever and it works to boot! But is it working around something that should be supplied in core?

Reply:
> But is it working around something that should be supplied in core?

That is absolutely the question and the answer (in my opinion) is absolutely.

The only right fix is for there to be a built-in mechanism to define one template for content and another for a query.  

HOWEVER this mechanism is exactly what has been happening in classic themes and has become the expectation.  From what I've read this behavior (there were a few mechanisms to achieve it) with front-page.php has been the standard way to achieve what we're achieving here for some time.

So while I absolutely believe that core SHOULD provide this, and we should get that on the roadmap, I also believe that until that lands this could be a pretty reasonable method to achieve the desired results.